cram_3d_world/cram_urdf_environment_manipulation/src/action-designators.lisp

```diff
@@ -59,7 +59,14 @@ NAME in the environment BTR-ENVIRONMENT."
     (-> (spec:property ?action-designator (:arm ?arm))
         (true)
         (man-int:robot-free-hand ?_ ?arm))
-    (spec:property ?action-designator (:distance ?distance))
```

Comment:
This is starting to be very copy pasty. Maybe you can look into merging the two designators.

Reply:
I will. But I think this would be better in a separate PR.